monitor/services/src/main/java/org/datacleaner/monitor/server/components/ComponentHandler.java

```diff
@@ -89,7 +91,6 @@
     public ComponentHandler(DataCleanerConfiguration dcConfiguration, String componentName) {
         _dcConfiguration = dcConfiguration;
         _componentName = componentName;
-        _stringConverter = new StringConverter(dcConfiguration);
```

Comment:
using non-final so it can be lazy-created in getter. Avoids creation of the string converter for simple cases, e.g. when property is simple string value.